configuration.revanced.json

```diff
@@ -145,6 +145,46 @@
 					}
 				}
 			}
+		},
+		{
+			"includes": {
+				"channels": [952946952348270626, 953965039105232906],
+				"match": [
+					"(?i)(((tg|telegram|downloaded|download|revanced|channel).*){3,})"
+				]
+			},
+			"excludes": {
+				"roles": [],
+				"match": ["(?i)discord, (?i)error", "(?i)problem", "(?i)get"]
+			},
+			"condition": {
+				"user": {
+					"server_age": 5
+				}
+			},
+			"response": {
+				"embed": {
+					"title": "Official ReVanced social media",
+					"description": "You may have encountered a fake channel, please check out the official ReVanced social media links which are located in <#954066838856273960> channel.",
+					"color": 14908858,
+					"fields": [],
```

Comment:
Add a "What now?" field and explain what to do now, such as leaving the Telegram channel, checking out the help and link channel, and so on.